jaimemendozadev-los-angeles.md

```diff
@@ -0,0 +1,17 @@
+##Los Angeles, CA
+
+### Restaurants
+
++ [Tsujita](https://www.yelp.com/biz/tsujita-la-artisan-noodle-los-angeles-2)
++ [In-N-Out Burger](http://www.in-n-out.com/)
```

Comment:
LOL, I'm personally an In N Out kind of guy.

Five Words: Four-by-four animal style. 👍 😄